src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -304,7 +304,20 @@ private String replacePlaceholders(String input, GuildMessageReceivedEvent event
                 .replace("\\~", "~") // get rid of escaped characters, since Minecraft doesn't use markdown
                 .replace("\\*", "") // get rid of escaped characters, since Minecraft doesn't use markdown
                 .replace("\\_", "_") // get rid of escaped characters, since Minecraft doesn't use markdown
-                .replace("%message%", message);
+                .replace("%message%", message)
+                .replace("%reply%", event.getMessage().getReferencedMessage() != null ? replaceReplyPlaceholders(DiscordSRV.config().getString("DiscordToMinecraftMessageReplyFormat"), event.getMessage().getReferencedMessage()) : "");
```

Comment:
Could (or probably should) use a LangUtil.Message instead of DiscordSRV.config().getString()